app/executor.php

```diff
@@ -508,12 +508,12 @@ function (string $runtimeId, array $vars, string $data, $timeout, $activeRuntime
             $functionStatus = ($statusCode >= 200 && $statusCode < 300) ? 'completed' : 'failed';
         
             Console::success('Function executed in ' . $executionTime . ' seconds, status: ' . $functionStatus);
-        
+
             $execution = [
                 'status' => $functionStatus,
                 'statusCode' => $statusCode,
-                'stdout' => \utf8_encode(\mb_substr($stdout, -16384)),
-                'stderr' => \utf8_encode(\mb_substr($stderr, -16384)),
+                'stdout' => \mb_strcut($stdout, 0, 1000000), // Limit to 1MB
```

Comment:
Can we get this number from config? Its something that jake did in his PR: https://github.com/appwrite/appwrite/pull/3160/files

Reply:
+1 ^

Reply:
It's fine here since this is the executor, and we will move it out of Appwrite eventually. So we cannot use the attributes from Appwrite in here.